test-clients/src/main/resource/contract/solidity/TransferAliased.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.8.12;
+
+import './AliasedOperator.sol';
+
+contract AliasedTransfer {
+    address aliasedOperator;
+
+    constructor() public {}
+
+    function initialize(address token) external returns (address operator) {
+        bytes memory bytecode = type(AliasedOperator).creationCode;
+        bytes32 salt = keccak256(abi.encodePacked(token));
+        assembly {
+            operator := create2(0, add(bytecode, 32), mload(bytecode), salt)
+        }
+        AliasedOperator(operator).initialize(token);
+        aliasedOperator = operator;
+    }
+
+    function transfer(address to, uint value) public {
+        AliasedOperator(aliasedOperator).transfer(to, value);
+    }
+
+    function associateOperator() public {
+        AliasedOperator(aliasedOperator).associate();
+    }
```

Comment:
We do not need those 2 functions. Ideally we should call the `AliasedOperator` contract directly and call the `transfer` there